Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Sharing/SpatialCoordinateSystemManager.cs

```diff
@@ -156,7 +159,14 @@ private void OnConnected(SocketEndpoint endpoint)
             participants[endpoint] = participant;
             participant.ShowDebugVisuals = showDebugVisuals;
 
-            ParticipantConnected?.Invoke(participant);
+            if (ParticipantConnected == null)
+            {
+                Debug.LogWarning("Participant created, but no connection listeners were found");
+                return;
+            }
```

Comment:
I think this will create warnings in the DSLR compositor, as nothing listens to this event in the Compositor scene. I think we shouldn't log a warning for this.